src/test/kotlin/com/codescene/jetbrains/util/SortDeltaFindingsTest.kt

```diff
@@ -5,30 +5,31 @@ import com.codescene.jetbrains.config.global.CodeSceneGlobalSettingsStore
 import com.codescene.jetbrains.config.global.MonitorTreeSortOptions
 import com.intellij.openapi.application.Application
 import com.intellij.openapi.application.ApplicationManager
-import com.jetbrains.rd.util.ConcurrentHashMap
 import io.mockk.every
 import io.mockk.mockk
 import io.mockk.mockkStatic
 import org.junit.Assert.assertEquals
 import org.junit.Before
 import org.junit.Test
+import java.util.Optional
+import java.util.concurrent.ConcurrentHashMap
 
 class SortDeltaFindingsTest {
     private lateinit var mockApplication: Application
     private lateinit var mockSettingsStore: CodeSceneGlobalSettingsStore
     private val mockDeltaResults = ConcurrentHashMap(
         mapOf(
             "aFile" to mockk<Delta>(relaxed = true) {
-                every { oldScore } returns 3.21
-                every { newScore } returns 5.32
+                every { oldScore } returns Optional.of<Double>(3.21)
+                every { newScore } returns Optional.of<Double>(5.32)
             },
             "bFile" to mockk<Delta>(relaxed = true) {
-                every { oldScore } returns 5.43
-                every { newScore } returns 9.32
+                every { oldScore } returns Optional.of<Double>(5.43)
+                every { newScore } returns Optional.of<Double>(9.32)
             },
             "cFile" to mockk<Delta>(relaxed = true) {
-                every { oldScore } returns 9.32
-                every { newScore } returns 1.93
+                every { oldScore } returns Optional.of<Double>(9.32)
+                every { newScore } returns Optional.of<Double>(1.93)
```

Comment:
Same as with previous explicit typing, you can optionally remove it here

```suggestion
                every { newScore } returns Optional.of(1.93)
```